apps/picsa-tools/monitoring-tool/src/app/components/form-item/form-item.component.html

```diff
@@ -1,9 +1,20 @@
-<mat-card appearance="raised" class="link" [routerLink]="['view', form._id]">
+<mat-card
+  appearance="raised"
+  class="link"
+  [routerLink]="isLocked ? null : ['view', form._id]"
+  (click)="handleFormClick($event)"
+  [class.processing]="isProcessing()"
+>
   <!-- Header -->
   <mat-card-header>
     <mat-card-title-group>
       <img mat-card-sm-image [src]="form.cover.icon" />
-      <mat-card-title>{{ form.title | translate }}</mat-card-title>
+      <mat-card-title>
+        {{ form.title | translate }}
+        <mat-icon *ngIf="isLocked" class="lock-icon">lock</mat-icon>
```

Comment:
nit(non-blocking): if you get a chance can you also refactor these and any other `*ngIf` statements you notice to use `@if`?
Not a huge issue either way, at some point I'll probably just run an automatic codegen to replace all, just good to get into the habit of it moving forwards